Tests/SonarQube.TeamBuild.PreProcessor.Tests/SonarWebServiceTest.cs

```diff
@@ -25,7 +25,7 @@ public class SonarWebServiceTest
         public void Init()
         {
             downloader = new TestDownloader();
-            ws = new SonarWebService(downloader, "http://localhost:9000");
```

Comment:
I replaced all the uses of http://localhost:9000 in this class with a different test value to make it easier to check where localhost:9000 is used in the product code (it isn't any longer).